src/redis-cli.c

```diff
@@ -893,10 +906,16 @@ static int cliSendCommand(int argc, char **argv, int repeat) {
     while(repeat--) {
         redisAppendCommandArgv(context,argc,(const char**)argv,argvlen);
         while (config.monitor_mode) {
+            in = 1;
```

Comment:
Can you explain this? Is this just so we can exit cleanly from `cliSendCommand` when we abort monitor? If so maybe a better name can be `monitor_aborted`?